src/components/CallView/CallView.vue

```diff
@@ -656,6 +662,8 @@ export default {
 
 			if (!this.screenSharingActive && this.speakers.length) {
 				this.sharedDatas[this.speakers[0].id].promoted = true
+			} else if (this.shownRemoteScreenPeerId) {
+				this.sharedDatas[this.shownRemoteScreenPeerId].promoted = true
```

Comment:
This part is also not clear to me, could you enplane the change here?

Reply:
When screensharing, the `promoted` attribute for the peer who is sharing wasn't updated. As we only pass the promoted model to the overlay, we won't have remote screenshare there. So I added those two lines to update the attribute.

Reply:
Nice, my solution was more complex 😀